.yarn/patches/@metamask-keyring-controller-npm-7.0.0-962008b200.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/dist/KeyringController.d.ts b/dist/KeyringController.d.ts
+index 82de83a7bb1ad14bb23f3b6274e0c4d5bb773382..86a09b3f604f6feb26e2c7edbdcb0abebd4bae20 100644
+--- a/dist/KeyringController.d.ts
++++ b/dist/KeyringController.d.ts
+@@ -1,10 +1,11 @@
+ import type { TxData, TypedTransaction } from '@ethereumjs/tx';
+-import { type MetaMaskKeyring as QRKeyring, type IKeyringState as IQRKeyringState } from '@keystonehq/metamask-airgapped-keyring';
++import type { MetaMaskKeyring as QRKeyring, IKeyringState as IQRKeyringState } from '@keystonehq/metamask-airgapped-keyring';
```

Comment:
Curious why these type imports were changed. Was this unintended? Or did the `import { type ___ }` syntax cause problems with the build system?

Reply:
It is intended, but it looks like it breaks only the lint step: https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/51596/workflows/800ab01e-5b6f-44c1-9312-934fdd12c838/jobs/1501250

Reply:
Ah I see. The TypeScript version used here is fairly old, that's probably why.

Reply:
Hmm. Should we add TypeScript `~4.6.3` as an optional peer dependency to all packages in the `core` repo?

Reply:
Not sure, since the package can still be used by JavaScript projects

Reply:
Maybe `>=4.6.3` instead? But yes that might be a good idea